src/AzureStack/Subscriptions/Subscriptions.Tests/SessionRecords/SubscriptionsManagementClient/TestCreateUpdateAndThenDeleteSubscription.json

```diff
@@ -0,0 +1,133 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "/offers?api-version=2015-11-01",
```

Comment:
The tenant wide /offers and /subscriptions calls are not the Microsoft.Subscriptions resource provider but are direct calls to Azure Stack/On-Prem ARM.

Reply:
Yes this is similar to the azure subscription client call at https://github.com/Azure/azure-rest-api-specs/blob/master/specification/subscription/resource-manager/Microsoft.Subscription/stable/2016-06-01/subscriptions.json#L91
Azure puts the /subscriptions call under SubscriptionClient, though the call is not going to be routed to Subscription RP.  I think it makes sense to group all the subscription operations under one client